Sora/Sora.cs

```diff
@@ -454,6 +454,12 @@ public void Connect(Config config)
             {
                 c.SetCompress(m.Compress.Value);
             }
+            if (m.Header != null)
+            {
+                var header = new SoraConf.Internal.DataChannel.Header();
+                header.content.AddRange(m.Header);
```

Comment:
初期化と同時に代入の方がいい感じになりそう。
```suggestion
                var header = new SoraConf.Internal.DataChannel.Header() {
                    content = m.Header,
                 };
```

Reply:
初期化と同時に代入するように修正します。